carmin-server/server/resources/helpers/executions.py

```diff
@@ -104,24 +104,16 @@ def inputs_file_path(username: str, execution_identifier: str) -> str:
         execution_identifier, INPUTS_FILENAME)
 
 
-def get_execution_as_model(username: str, identifier: str,
-                           db_session) -> (Execution, ErrorCodeAndMessage):
-    execution_db = db_session.query(ExecutionDB).filter_by(
-        identifier=identifier).first()
-
-    if not execution_db:
-        return None, EXECUTION_NOT_FOUND
-
-    inputs, errors = load_inputs(username, execution_db.identifier)
-    if errors:
-        return None, errors
-
+def get_execution_as_model(username: str, db_execution):
```

Comment:
I believe this should take into account the situation where db_execution is None. This could happen in comment (1)

Reply:
Also, please revert to execution_db as discussed.